src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/TypeEditor.test.tsx

```diff
@@ -474,6 +477,34 @@ describe("TypeEditor Component", () => {
       screen.getByTestId("uri-input-field-DiagnosticReport.presentedForm.uri")
     ).toBeInTheDocument();
   });
+  it("Should render URI component ( invalid input validation)", async () => {
+    const handleChange = jest.fn();
+
+    render(
+      <FormikProvider value={mockFormik}>
+        <TypeEditor
+          type={`uri`}
+          required={true}
+          resource={null}
+          value={`urn:oid:2.16.840.1.113883.6.238`}
+          onChange={handleChange}
+          structureDefinition={null}
+          label={"DiagnosticReport.presentedForm.uri"}
+          canEdit={true}
+          parentStructureDefinition={null}
+        />
+      </FormikProvider>
+    );
+    const inputField = screen.getByTestId(
+      "uri-input-field-DiagnosticReport.presentedForm.uri"
+    );
+    expect(inputField).toBeInTheDocument();
+    await act(async () => {
+      userEvent.type(inputField, "urn:oid:AA");
+    });
```

Comment:
Seems `act` is deprecated, maybe change to `waitFor`?

Reply:
Never mind, I'm still using the old import in my test file. Using  act in "testing-library/react" resolves the deprecation warning in my local.